brainstormforce/ultimate-addons-for-gutenberg
# WIP UAG-438 Update single unit for all device type.

### Description

- https://ua-gutenberg.atlassian.net/browse/UAG-438

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- drag and drop each block and check spacing control.
- Check single unit for all device type

### Checklist:
